lib/core/src/sdk.rs

```diff
@@ -1226,14 +1249,34 @@ impl LiquidSdk {
                             },
                         )?;
                         let receiver_amount_sat = asset_metadata.amount_to_sat(receiver_amount);
-                        let fees_sat = self
+                        let fees_sat_res = self
                             .estimate_onchain_tx_or_drain_tx_fee(
                                 receiver_amount_sat,
                                 &liquid_address_data.address,
                                 &asset_id,
                             )
-                            .await?;
-                        (asset_id, receiver_amount_sat, fees_sat)
+                            .await;
+                        let fees_res = self
+                            .payjoin_service
+                            .estimate_payjoin_tx_fee(&asset_id, receiver_amount_sat)
+                            .await;
```

Comment:
This requires a network request. My concern is about slowing the prepare.
Did you consider adding an explicit flag to request that? A field on the PrepareSendPaymentRequest for example?
My assumption is that lots (perhaps most) of the users won't need that.

Reply:
I added an `estimate_asset_fees` field to the PayAmount::Asset enum [29b8e3d](https://github.com/breez/breez-sdk-liquid/pull/779/commits/29b8e3d8fb9ffc055d807560ebcb92011bec9546)